client/signup/steps/woocommerce-install/hooks/use-woop-handling/index.ts

```diff
@@ -152,9 +141,7 @@ export default function useEligibility( siteId: number ): EligibilityHook {
 	);
 
 	// The site requires upgrading when the feature is not active and available.
-	const requiresUpgrade = Boolean(
-		eligibilityNoProperPlan && ! isWoopFeatureActive && hasWoopFeatureAvailable.length
-	);
+	const requiresUpgrade = Boolean( ! isWoopFeatureActive && hasWoopFeatureAvailable.length );
```

Comment:
Removing the plan check here. 

@lsl @Rolilink @obenland @DustyReagan Do any of you happen to remember why this was needed in addition to the feature check? Did we add it before we included upgrading in the flow?

There's one other reference to business plans here:
https://github.com/Automattic/wp-calypso/blob/1c435b43df496119415b249639d5ff5dd2d6b2b2/client/signup/steps/woocommerce-install/hooks/use-woop-handling/index.ts#L27
That ensures that the plan check is skipped for transfers, though, so it shouldn't need a change.

Reply:
I don't but @retrofox might?

Reply:
🤔 I think the definition of the woop feature changed. At first the Woop feature was going to mean which sites had the ability to go through the woop install workflow. Like, if we rolled out gradually (instead we opened it up to all plans at once). So in the former case a sub-business plan could have the ability to go the through the woop workflow (thus having the woop feature available but not active), and we'd need to know if they already had a business plan or not? Something along those lines.

Reply:
Thanks! 🙌 That would explain it.

Reply:
Not sure, would need to dig in, as long as you tested all the scenarios starting from the landing page it should be fine.

- non business plan simple site
- business plan simple site 
- business plan atomic site